helmfile.d/0400.kube-prometheus.yaml

```diff
@@ -110,6 +110,8 @@ releases:
                   ### Required: KUBE_PROMETHEUS_ALERT_MANAGER_SLACK_CHANNEL; e.g. #alerts-staging
                   channel: '{{ env "KUBE_PROMETHEUS_ALERT_MANAGER_SLACK_CHANNEL" }}'
                   send_resolved: true
+          templates:
+            - ./*.tmpl
```

Comment:
where do we have those `.tmpl` files?

Reply:
Here
https://github.com/cloudposse/helmfiles/pull/48/files#diff-e6ab6531a3930583484f9a763ca1fd7b
https://github.com/cloudposse/helmfiles/pull/48/files#diff-4d017cfa59e63b8a332ebe7374104e48R163